src/lang/en-CA.json

```diff
@@ -1,356 +1,356 @@
 {
 	"account.account-settings": "Account Settings",
-	"account.forgot-password": "Forgot Password?",
-	"account.no-account": "Don't have an account?",
-	"account.request-password-reset": "Enter your email address and we will send you an email to reset your password.",
-	"account.sign-out": "Sign Out",
-	"account.sign-up": "Sign Up",
-	"account.sign-in": "Sign In",
 	"account.already-have-account": "Already have an account?",
+	"account.already-organisation-affiliate": "You are already affiliated with [name]",
+	"account.finish-registration": "Finish Registration",
+	"account.forgot-password": "Forgot Password?",
 	"account.join-organisation": "Join Your Organization",
```

Comment:
why are all these organization references spelled organisation?

Reply:
I noticed that there are several places in the language files where the key has organization spelled as 'organisation' such as in the example you mentioned. Being that it is the key and not the value of what gets displayed in the UI, I did not update it to 'organization'. 

I can change all of the keys for consistency. BTW, I'll be making more changes in a future PR. I can pick up this change request then.